golang/cmd/aws_key_gen/script.go

```diff
@@ -0,0 +1,122 @@
+// MIT License
```

Comment:
Sorry, we cannot give you the copyright for this file due to Google's policy towards contributions. What you can do is add in the Google Apache license and then add an extra line saying that you are the original author of this file that links to your website.

```
/* Copyright 2021 The Subscribe with Google Authors. All Rights Reserved.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *      http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS-IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 * 
 * Original author: Apoorv Mote https://apoorv.blog
 */
```

Reply:
No worries. I have updated license. Thanks for allowing me to add link to my website.